packages/mui-base/src/PopperUnstyled/PopperUnstyled.js

```diff
@@ -40,6 +40,7 @@ const defaultPopperOptions = {};
 const PopperTooltip = React.forwardRef(function PopperTooltip(props, ref) {
   const {
     anchorEl,
+    component: Root = 'div',
```

Comment:
@michaldudak Could you review this one? 

My intention is to use Joy's Sheet component instead of a plain div.